lib/utils/resolve-circular.ts

```diff
@@ -0,0 +1,11 @@
+import copy from 'fast-copy'
+import mixinStringifySafe from '../mixins/stringify-safe'
+import resolveResponse from 'contentful-resolve-response'
+
+export function resolveCircular (data, { resolveLinks, removeUnresolved }) {
+  const wrappedData = mixinStringifySafe(copy(data))
```

Comment:
Actually `resolveResponse` [already makes a copy](https://github.com/contentful/contentful-resolve-response/blob/master/index.js#L114) so I think we can skip that copy as well.

Reply:
@TimBeyer yay, that was the last ref to `fast-copy` :)